bin/deploy-eks

```diff
@@ -184,6 +188,14 @@ if [[ $application_name == 'fb-service-token-cache' ]]; then
   helm_command="${helm_command} --set bearer_token=${bearer_token}"
 fi
 
+if [[ $application_name == 'fb-av' ]]; then
```

Comment:
can we have a comment above here please describing the need to deploy out to the two namespaces and two different apps will be accessing the fb-av

Reply:
yep can do